packages/@lightningjs/ui-components/src/components/Shadow/Shadow.js

```diff
@@ -75,7 +77,7 @@ export default class Shadow extends Base {
               h: this.h - holepunchError,
               x: shadowSize + holepunchError / 2,
               y: shadowSize + holepunchError / 2,
-              radius: this.style.radius
+              radius: radius
```

Comment:
you could simplify this to just `radius` since the key and variable have the same name